src/demo/pages/PivotPage/PivotPage.tsx

```diff
@@ -39,6 +41,9 @@ export class PivotPage extends React.Component<any, any> {
         <ExampleCard title='Large link size' code={ PivotLargeExampleCode }>
           <PivotLargeExample />
         </ExampleCard>
+        <ExampleCard title='Item Count' code={ PivotItemCountExampleCode }>
```

Comment:
super minor nit but good lesson for consistency, try to keep titles Sentence cased (count should be lower cased)